src/Helpers/Test.php

```diff
@@ -5,8 +5,22 @@
 use Illuminate\Database\Eloquent\Model;
 
 if (! function_exists('randomOrCreate')) {
-    function randomOrCreate(string $className): Model
+	/**
+	 * Ger random model or create model using factory.
+	 *
+	 * @param  string|Model $classNameOrModel
```

Comment:
@drewroberts FYI - that's the case where docblock must be used instead of type-hinting.

Ref.: https://www.php.net/manual/en/language.types.declarations.php#language.types.declarations.union

Reply:
Thanks for pointing that out